README.md

```diff
@@ -0,0 +1,91 @@
+# UrContacts
+> A contact list application
+
+UrContacts is a React contact list app built with full CRUD capabilities. It uses json-server to locally host a JSON file database on the backend, where your data will persist.
+
+## Installation and Build
+
+1. `npm init -y`
+
+2. This app was built with Vite. While you may use whatever React bootstrapping tool you prefer to get the app running, Vite has some useful features built-in, such as TypeScript and a server: [Vite][vite]. Instructions from here on out will assume you have some way a spinning a server locally.
+
+3. If you use a bootstrapping tool other than Vite, make sure it has TypeScript baked in, or install TypeScript separately: [TypeScript][typescript]
+
+4. `npm install --save json-server react-router-dom` 
+
+5. Add your server start scripts to your `package.json`:
+
+    "scripts": {
+        // .. 
+        "server:dev": "json-server --watch ./data/db.json",
+        "start": "json-server ./data/db.json --static ./dist"
+        //..
+    }
+
+6. To run your server: `npm run server:dev`
+
+7. In a separate terminal, start your React app using Vite: `npm run dev`
+
+### Additional Notes
+UrContacts uses React Router to route to various URLs in on your browser: 
+
+/                               => Home page
+/contacts                       => contact list
+/contacts/{contactID}           => contact by id
+/create                         => create new contact
+/contacts/{contactID}/update    => update contact
+
+--
+
+For more information about json-server, consult the docs here: [npm: json-server][json-server]  
+
+## Information
+
+UrContacts provides functionality for creating, updating, deleting, and searching for contacts in a list stored as a JSON file on a locally hosted json-server. The app uses async calls to the json-server using the fetch API, and data that you add, update, or delete persist in the json-server in the `db.json` file. The app uses the React Router package to create routes to different application features while maintaining a single-page application experience.
```

Comment:
```suggestion
UrContacts provides functionality for creating, updating, deleting, and searching for contacts in a list stored as a JSON file on a locally hosted json-server. The app uses async calls to the json-server using the fetch API, and data that you add, update, or delete persist in the json-server in the `db.json` file. The app uses the React Router library to create routes to different application features while maintaining a single-page application experience.
```

Reply:
@dyazdani  did you miss this one? (you can hit "Commit suggestion" button if you like the suggestion btw)

Reply:
Got it. done.